packages/backend/src/graphql/resolvers/walletAddressKey.ts

```diff
@@ -4,10 +4,51 @@ import {
   WalletAddressKey as SchemaWalletAddressKey,
   Alg,
   Kty,
-  Crv
+  Crv,
+  WalletAddressResolvers,
+  SortOrder
 } from '../generated/graphql'
 import { ApolloContext } from '../../app'
 import { WalletAddressKey } from '../../open_payments/wallet_address/key/model'
+import { Pagination } from '../../shared/baseModel'
+import { getPageInfo } from '../../shared/pagination'
+
+export const getWalletAddressKeys: WalletAddressResolvers<ApolloContext>['walletAddressKeys'] =
+  async (
+    parent,
+    args,
+    ctx
+  ): Promise<ResolversTypes['WalletAddressKeyConnection']> => {
+    if (!parent.id) {
+      throw new Error('missing wallet address id')
+    }
+
+    const walletAddressKeyService = await ctx.container.use(
+      'walletAddressKeyService'
+    )
+
+    const { sortOrder, ...pagination } = args
+    const order = sortOrder === 'ASC' ? SortOrder.Asc : SortOrder.Desc
+
+    const walletAddressKeys = await walletAddressKeyService.getPage(
+      parent.id,
+      pagination,
+      order
+    )
+    const pageInfo = await getPageInfo({
+      getPage: (pagination_?: Pagination, sortOrder_?: SortOrder) =>
+        walletAddressKeyService.getPage(parent.id!, pagination_, sortOrder_),
```

Comment:
I'm definitely confused why `parent.id` is defined after explicitly checking for it, but I guess that's some weird TS shenanigans 🤷‍♂️

Reply:
@mkurapov apparently `parent.id` is of type `(property) id?: string | undefined`. And even though I am checking if `parent.id` is undefined and then throwing error, without this explicitly marking as defined, it doesn't go through.

I have also updated bruno collection